packages/lib/services/plugins/utils/isCompatible/index.test.ts

```diff
@@ -0,0 +1,105 @@
+import { AppType } from '../../api/types';
```

Comment:
```suggestion
import { AppType } from '../../../models/Setting';
```

Please consistently import `AppType` from the same place — currently, some places use `models/Setting/AppType` and others use `plugins/api/types/AppType`.

It could, alternatively, make sense to use an `isMobile` boolean variable, but doing so would make adding a new platform (e.g. web or CLI) more difficult in the future.